addons/website_sale/models/website.py

```diff
@@ -243,9 +242,6 @@ def sale_get_order(self, force_create=False, code=None, update_pricelist=False,
 
             request.session['sale_order_id'] = sale_order.id
 
-            if request.website.partner_id.id != partner.id and partner.last_website_so_id.id != sale_order.id:
-                partner.write({'last_website_so_id': sale_order.id})
```

Comment:
:+1: This was doing nothing right?
was last_website_so_id a non computed field in the past or had a inverse method?

Reply:
yes, nothing since we replace it by compute field without inverse function